cfme/services/catalogs/cloud_catalog_item.py

```diff
@@ -53,45 +57,25 @@
         ('long_desc', "//textarea[@id='long_description']")
     ])
 
+match_page = partial(match_location, title='Catalogs', controller='catalog')
 
-def _all_catalogitems_add_new(context):
-    accordion.tree('Catalog Items', 'All Catalog Items')
-    tb_select('Add a New Catalog Item')
-    provider_type = context['provider_type']
-    sel.select("//select[@id='st_prov_type']", provider_type)
-
-
-def _all_catalogbundle_add_new(context):
-    sel.click("//div[@id='sandt_tree_div']//td[normalize-space(.)='All Catalog Items']")
-    tb_select('Add a New Catalog Bundle')
-
-
-nav.add_branch(
-    'services_catalogs',
-    {'catalog_items': [nav.partial(accordion.click, 'Catalog Items'),
-        {'catalog_item_new': _all_catalogitems_add_new,
-         'catalog_item': [lambda ctx: accordion.tree('Catalog Items', 'All Catalog Items',
-                                ctx['catalog'], ctx['catalog_item'].name),
-                          {'catalog_item_edit': nav.partial(tb_select,
-                                                            "Edit this Item")}]}]})
-
-
-class Template(Pretty):
-    pretty_attrs = ['name']
 
-    def __init__(self, name):
-        self.name = name
+def nav_to_all():
```

Comment:
Since this file contains a single class, I would just put this inside ItemAll.step(). We started pulling it into a function when we had multiple 'All' classes where a file defined more than one class.